### ISSUE TITLE
First Try-> total crach of the module

### ISSUE THREAD

**Issue by GrosBob** [Normalised/ProkDrumsEditor#28]
My Prok module (BD) was working perfectly fine after building (the three others are not finished yet). Great sound, perfect.
My tried the PDEditor for the first time (on OSX 10.13.6) seems to kill my module.
- while pluged to the case power bus, I plugged the usb cable directly to the computer.
- leds turnd off
- when I launched th PDEditor, the module was not reconized (it didin't evec apprear in the USB device list on the computer)
- After turning off the case power, unplugging the USB cable, the module do not boot anymore. No led are showing up.

Help.. what can I don to have it back working? 
Is ther a way to reset it back easily??
I begin to be scared if this happens later on using the software editors.

**Comment by Thonk-Steve** [comment]
Hi,

Thanks for posting. First question, did you cut the trace on the teensy as
per step 19 on the build document?

http://thonk.co.uk/documents/prok/Prok%20Drums%20Build%20Doc%20v1.0.pdf

Regards

Steve
Thonk

**Comment by GrosBob** [comment]
Hi,

I did cut the trace on the PCP.
Whats' strage is that I played with the module. It was working

**Comment by Thonk-Steve** [comment]
Have you confirmed with a multimeter that the trace is definitely cut?

Was it working ok until the moment you plugged in the USB?

**Comment by GrosBob** [comment]
Yes. I confirmed with a multimeter. And yes, the module was working perfectly before I plugged it to USB.
When I plugged it, leds tuned off, and never light up again

**Comment by Thonk-Steve** [comment]
Ok it kind of sounds like you possibly had a power spike over USB that has
blown something. Don't plug anything else into that USB port or with that
USB cable.  This can sometimes be due to damaged cables. This probably
isn't catastrophic but might require replacing a component or two.

I would suggest the following steps:

1) Send us photos of both sides of the Prok PCBs and teensy (6 photos in
total)
3) Use your multimeter in DC voltage mode to double check your Eurorack PSU
is putting out approx +12v and -12v
4) Solder the headers on a teensy from one of the other modules, e.g. CP
and mount on the BD module, don't plug in USB!
4) See if the module runs the CP teensy ok. This is a safe thing to do.

Steve

**Comment by GrosBob** [comment]
Thank tyou.
I'll do so and get back to you

**Comment by GrosBob** [comment]
Dear Steve,

I finally manage to sort out my problem. A little bit strange as you’ll see:

- First: All the 4 Teensy are properly working!! They’ve been successfully updated to v5.x and the editor is working great and all the 4 Teensys are perfectly working. Good point.
But I had to install ProkDrumCentral to another computer running OSX 10.15.7, and it worked like a charm in updating the firmwares to v5.x. Then, going back to my “music” computer (running OSX 10.13.6), everything was perfectly fine. ProkModularEditor is recognising the modules, patches can be tweaked, saved and export. The same for ProkDrumCentral (on 10.13.6). I could swap firmwares, flash v3.x version, revert to 5.x and so on. In other words, everything in fine and the modules are really wonderful !! I love them, and the Klonk is awesome!

- But it would have been too perfect. While I was trying to find out my initial problem (BD module working, but Teensy not recognised by the computer) I might have done a mistake moving the module out of the case to plug the USB cable as it din’t work after this (very weird, but this is my only explanation). The bottom PCB died… The regulator fried.. I first thought of a reverse voltage when plug it back, but no. I checked carefully somme critical components on that board: the 2 1N5819 were intacts and working fine and the 10uF caps didn’t bow.  I rely do not understand. I only notice something strange, a burn on the PCB (red arrow on the pict) on the OUT pad of the +12v Diode. I couldn’t find the +12v on the TL072. But It doesn’t make sense to me. From that I didn’t check the OpAmps.. They may have also fryed.

- As a summary, I burned one bottom PCB and several component on it. And it is totally my fault!! But the Control PCB is working fine (tested on another module)
So my question is do you sell the PCB and all the components for the bottom board only. No Teensy, no panel, no button?? So I’ll be able to finish the forth and last Prok Module.

I also would like to by two others kits from you: 
- MusicThing Modular Turing Machine Mk2
- MusicThing Modular Turing machine Expansion for Mk2

Could you give me a quote for all of this? (Including the best solution you have for re-making a full Prok bottom board)
And how to proceed to the order.

Thanks a lot in advance, and rely sorry for bothering you with this,

Looking forward to reading you

Best regards

Eric

**Comment by Thonk-Steve** [comment]
Ok thanks, drop our sales team a line on support@thonk.co.uk and they can
arrange 👍
